llvm/lib/Target/RISCV/RISCVProcessors.td

```diff
@@ -381,3 +381,20 @@ def XIANGSHAN_NANHU : RISCVProcessorModel<"xiangshan-nanhu",
                                             TuneZExtHFusion,
                                             TuneZExtWFusion,
                                             TuneShiftedZExtWFusion]>;
+
+def SPACEMIT_X60 : RISCVProcessorModel<"spacemit-x60",
+                                       NoSchedModel,
+                                       !listconcat(RVA22S64Features,
+                                       [FeatureStdExtV,
+                                        FeatureStdExtSvnapot,
+                                        FeatureStdExtZbc,
+                                        FeatureStdExtZbkc,
+                                        FeatureStdExtZfh,
+                                        FeatureStdExtZicond,
```

Comment:
Looking at e.g. zicond, I don't see anything in the docs which claim support for this extension.  ruapu appears to work via running an instruction and checking for faults - this could be a dangerous thing to rely on if the CPU e.g. incorrectly implements some extension, or doesn't implement the reserved opcodes the way the tool expects.

I suggest that we cut down this list to include *only* those things which are specified.  We can expand that list later based on changes to documentation and/or communication with vendor.

Reply:
Thanks for your review, it really is more reliable to add support based on document. The supported extensions are listed in 2.1.2 of this [document](https://developer.spacemit.com/#/documentation?token=BWbGwbx7liGW21kq9lucSA6Vnpb), including zicond. If you have any questions about other extensions, please let me know so that I can confirm and update the document. Finally, we can review this patch based on the document.

Reply:
> The supported extensions are listed in 2.1.2 of this [document](https://developer.spacemit.com/#/documentation?token=BWbGwbx7liGW21kq9lucSA6Vnpb), including zicond. I

@sunshaoce  We should adjust the patch description to reference this document.  If you have anything you think is supported which is not on the documented list, please highlight it so that @zqb-all can clarify docs.

@zqb-all Thanks!

Reply:
Updated the description.